.gitignore

```diff
@@ -1,6 +1,12 @@
 *.py[cod]
 __pycache__
 
+# temp files from docs build
+doc/auto_gallery
+doc/example.nc
+doc/gallery
```

Comment:
`doc/gallery` is part of the git tree

(https://github.com/pydata/xarray/tree/master/doc/gallery)